mne/minimum_norm/tests/test_inverse.py

```diff
@@ -349,15 +352,16 @@ def test_localization_bias_loose(bias_params_fixed, method, lower, upper,
     ('dSPM', 40, 45, {}, 0.8),
     ('sLORETA', 90, 95, {}, 0.8),
     ('eLORETA', 90, 95, dict(method_params=dict(force_equal=True)), 0.8),
-    ('eLORETA', 100, 100, {}, 0.8)])
+    ('eLORETA', 100, 100, {}, 0.8),
+])
 def test_localization_bias_free(bias_params_free, method, lower, upper,
                                 kwargs, depth):
     """Test inverse localization bias for free minimum-norm solvers."""
     evoked, fwd, noise_cov, _, want = bias_params_free
     inv_free = make_inverse_operator(evoked.info, fwd, noise_cov, loose=1.,
                                      depth=depth)
     loc = apply_inverse(evoked, inv_free, lambda2, method,
-                        pick_ori='vector', **kwargs).data
+                        pick_ori='vector', verbose='debug', **kwargs).data
```

Comment:
revert?

Reply:
The `'debug'` shouldn't hurt during runs, and helps when they do fail